cmp/dimensionchooser/DimensionChooserModel.js

```diff
@@ -63,8 +66,10 @@ export class DimensionChooserModel {
      *      form supports value, label, and leaf keys, where `leaf: true` indicates that the
      *      dimension does not support any further sub-groupings.
      * @param {string[]} [c.initialValue] - initial dimensions if history empty / not configured.
-     * @param {string} [c.historyPreference] - preference key used to persist the user's most
-     *      recently selected groupings for easy re-selection.
+     * @param {string} [c.preference] - preference key used to persist the user's last value
+     *      and most recently selected groupings for easy re-selection.
+     * @param {boolean} [c.persistValue] - persist last value to preference.
```

Comment:
Would it be most straightforward to 

* always persist value, if there is a preference
* persist (and show) the history based on maxHistoryLength (e.g., if <=0 no need to show or store the history)